recipe/meta.yaml

```diff
@@ -24,22 +23,26 @@ test:
 outputs:
   - name: mdahole2-base
     script: build_base.sh
+    build:
+      noarch: python
     requirements:
       host:
-        - python >=3.9
+        - python >=3.9,<3.13
```

Comment:
Is `<3.13` extra precaution (because we haven't tested on >= 3.12 and cannot guarantee it's working) or is it absolutely required? Just curious.

Reply:
Just a precaution but also something I left whilst playing around with things - tbh this is guarded by the MDA pin, so we're probably ok to remove it